### PR TITLE
Feature/buddy into development

### PR DESCRIPTION
**Pull request by dewarian**
Merging the buddies to development for further development.

image

### CODE REVIEW

File: frontend/components/buddyCard/BuddyCard.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          <span className={styles.pointer}><img src="/icons/chevron-icoon.svg" alt="" /></span>
+        </section>
+      </article>
+    </Link>
+    </>
+  )
+}
+
+/**
+ * @title getServerSideProps
```
Comment (sjagoori) [inline frontend/components/buddyCard/BuddyCard.js:31]: deze mogen weg

File: frontend/components/buddyCard/BuddyCard.module.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/* .card section > * {
```
Comment (sjagoori) [inline frontend/components/buddyCard/BuddyCard.module.css:1]: comments eruit

File: frontend/components/buddyList/BuddyList.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import BuddyCard from '@/components/buddyCard/BuddyCard';
+import styles from './BuddyList.module.css';
+
+// Needs to be removed, dummy data.
```
Comment (sjagoori) [inline frontend/components/buddyList/BuddyList.js:4]: formatting needed

**Review by sjagoori — CHANGES_REQUESTED** [review]

File: frontend/components/buddyCard/BuddyCard.module.css (hunk 2 above)
Comment (dewarian) [reply frontend/components/buddyCard/BuddyCard.module.css]: Fixed in commit feae509

**Review by sjagoori — APPROVED** [review]

**Pull request merged**